### ISSUE TITLE
Most recent Docker image is no longer working

### ISSUE THREAD

**Issue by chrism** [savonet/liquidsoap#2720]
After updating the Docker image from [v2.1.0](https://hub.docker.com/layers/savonet/liquidsoap/v2.0.7/images/sha256-440b61fc2249b8158da18c14c5893c22b7aa3ef80815f3de684b196df38794a3?context=explore) to [v2.1.2](https://hub.docker.com/layers/savonet/liquidsoap/v2.1.2/images/sha256-962555716ee6272061ecb41786f03f95e31599d160e224ad384d96a7a5966813?context=explore) there is an error message on a  fresh Digital Ocean droplet from their [official Docker image](https://marketplace.digitalocean.com/apps/docker) (Docker `19.03.12` on Ubuntu `20.04`).

```
Fatal error: exception Sys_error("Thread.create: Operation not permitted")
```

This is a minimal test case which works locally (on a mac), but fails when deployed.

Liquidsoap script which emits a sine to Icecast
```
#!/usr/bin/liquidsoap

settings.init.allow_root.set(true)
settings.log.level.set(5)

def generate_source() =
  radio = sine()
  radio
end

output.icecast(
  %mp3(
    bitrate=192,
    id3v2=true
  ),
  name = "Test Radio",
  mount = "test",
  password = getenv("ICECAST_SOURCE_PASSWORD"),
  host = "icecast",
  port = 8000,
  encoding = "UTF-8",
  generate_source()
)
```

And a `docker-compose.yml` file which uses the [Liquidsoap](https://hub.docker.com/r/savonet/liquidsoap) and [Icecast](https://hub.docker.com/r/infiniteproject/icecast) Docker images
```
version: '3'

services:
  icecast:
    image: "infiniteproject/icecast"
    restart: always
    ports:
      - "${ICECAST_PORT:-8000}:8000"
    environment:
      - ICECAST_SOURCE_PASSWORD
      - ICECAST_ADMIN_PASSWORD
      - ICECAST_RELAY_PASSWORD
      - ICECAST_HOSTNAME=test.radio
      - ICECAST_MAX_SOURCES=10
      - ICECAST_LOCATION=Paris
      - ICECAST_ADMIN_EMAIL=info@test.radio

  liquidsoap:
    image: "savonet/liquidsoap:v2.1.0"
    restart: always
    command: ["/script.liq"]
    ports:
      - 7001:7001
    environment:
      - ICECAST_SOURCE_PASSWORD
    depends_on:
      - icecast
    volumes:
      - music:/music:ro
      - ./script.liq:/script.liq:ro

volumes:
  music:
```

The version above works both locally and on a droplet.

But after updating the Liquidsoap image from `savonet/liquidsoap:v2.1.0` to `savonet/liquidsoap:v2.1.2` there is this error message.

```
Fatal error: exception Sys_error("Thread.create: Operation not permitted")
```

- OS: Ubuntu `20.04`
- Version:`2.1.2`

**Comment by toots** [comment]
Thanks for this report. What architecture are you using?

**Comment by chrism** [comment]
Hi @toots you're welcome–sorry but I'm out of my depth with hardware questions.

It's a basic Digital Ocean shared CPU droplet but I couldn't really find anything more precise from their [help page](https://docs.digitalocean.com/products/droplets/concepts/choosing-a-plan/)

If there are any specific details you need and you could aid me by explaining how I could find them out then I'm happy to do whatever I can.

**Comment by toots** [comment]
We build our images for multiple architectures, amd54, arm 64 and 32 bits. We have had issues in the past with the arm 32bit image (the build process is a little more tricky). When do run the docker image `savonet/liquidsoap:v2.1.2`, docker pulls the image corresponding to your actual architecture.

I have tested our arm 32 and 64 bit image as well as the amd64 images and they appear to work properly. I would suggest to look at your docker configuration to see if something might off, starting with security access/configuration.

Closing this for now, please feel free to re-open or report again if you see something coming from us.

Good luck!

**Comment by vitoyucepi** [comment]
Hi @chrism,

Could you post some information about your DO instance?
The `docker info` output will be good for the beginning.

For me, this issue sounds like MariaDB/mariadb-docker#434.
For the first time, I used `docker-ce=20.10.21`, and there was no problem there, but then I downgraded to `docker-ce=20.10.7`.
I can reproduce this issue with the old docker version.
I think that the minimum required version is `docker-ce=20.10.10`. Setting `seccomp` to `unconfined` will be fine too.
```yml
  liquidsoap:
    image: "savonet/liquidsoap:v2.1.2"
    restart: always
    command: ["/script.liq"]
    ports:
      - 7001:7001
    environment:
      ICECAST_SOURCE_PASSWORD: hackme
    depends_on:
      - icecast
    security_opt:
      - seccomp:unconfined
    volumes:
      - music:/music:ro
      - ./script.liq:/script.liq:ro
```

<details>
<summary>Details</summary>

I used ubuntu 20.04 [cloud image](https://cloud-images.ubuntu.com/focal/current/) and a *cloud config*.
```yaml
users:
  - default

system_info:
  default_user:
    name: ubuntu
    lock_passwd: false
    plain_text_passwd: ubuntu

package_update: true
package_upgrade: true

apt:
  primary:
    - arches: [default]
      uri: http://archive.ubuntu.com/ubuntu
  security:
    - arches: [default]
      uri: http://security.ubuntu.com/ubuntu
  sources_list: |
    deb $PRIMARY $RELEASE main universe restricted multiverse
    deb $PRIMARY $RELEASE-updates main universe restricted multiverse
    deb $PRIMARY $RELEASE-backports main universe restricted multiverse
    deb $SECURITY $RELEASE-security main universe restricted multiverse
  sources:
    docker:
      source: deb [signed-by=$KEY_FILE] https://download.docker.com/linux/ubuntu $RELEASE stable
      key: |
        -----BEGIN PGP PUBLIC KEY BLOCK-----
        
        mQINBFit2ioBEADhWpZ8/wvZ6hUTiXOwQHXMAlaFHcPH9hAtr4F1y2+OYdbtMuth
        lqqwp028AqyY+PRfVMtSYMbjuQuu5byyKR01BbqYhuS3jtqQmljZ/bJvXqnmiVXh
        38UuLa+z077PxyxQhu5BbqntTPQMfiyqEiU+BKbq2WmANUKQf+1AmZY/IruOXbnq
        L4C1+gJ8vfmXQt99npCaxEjaNRVYfOS8QcixNzHUYnb6emjlANyEVlZzeqo7XKl7
        UrwV5inawTSzWNvtjEjj4nJL8NsLwscpLPQUhTQ+7BbQXAwAmeHCUTQIvvWXqw0N
        cmhh4HgeQscQHYgOJjjDVfoY5MucvglbIgCqfzAHW9jxmRL4qbMZj+b1XoePEtht
        ku4bIQN1X5P07fNWzlgaRL5Z4POXDDZTlIQ/El58j9kp4bnWRCJW0lya+f8ocodo
        vZZ+Doi+fy4D5ZGrL4XEcIQP/Lv5uFyf+kQtl/94VFYVJOleAv8W92KdgDkhTcTD
        G7c0tIkVEKNUq48b3aQ64NOZQW7fVjfoKwEZdOqPE72Pa45jrZzvUFxSpdiNk2tZ
        XYukHjlxxEgBdC/J3cMMNRE1F4NCA3ApfV1Y7/hTeOnmDuDYwr9/obA8t016Yljj
        q5rdkywPf4JF8mXUW5eCN1vAFHxeg9ZWemhBtQmGxXnw9M+z6hWwc6ahmwARAQAB
        tCtEb2NrZXIgUmVsZWFzZSAoQ0UgZGViKSA8ZG9ja2VyQGRvY2tlci5jb20+iQI3
        BBMBCgAhBQJYrefAAhsvBQsJCAcDBRUKCQgLBRYCAwEAAh4BAheAAAoJEI2BgDwO
        v82IsskP/iQZo68flDQmNvn8X5XTd6RRaUH33kXYXquT6NkHJciS7E2gTJmqvMqd
        tI4mNYHCSEYxI5qrcYV5YqX9P6+Ko+vozo4nseUQLPH/ATQ4qL0Zok+1jkag3Lgk
        jonyUf9bwtWxFp05HC3GMHPhhcUSexCxQLQvnFWXD2sWLKivHp2fT8QbRGeZ+d3m
        6fqcd5Fu7pxsqm0EUDK5NL+nPIgYhN+auTrhgzhK1CShfGccM/wfRlei9Utz6p9P
        XRKIlWnXtT4qNGZNTN0tR+NLG/6Bqd8OYBaFAUcue/w1VW6JQ2VGYZHnZu9S8LMc
        FYBa5Ig9PxwGQOgq6RDKDbV+PqTQT5EFMeR1mrjckk4DQJjbxeMZbiNMG5kGECA8
        g383P3elhn03WGbEEa4MNc3Z4+7c236QI3xWJfNPdUbXRaAwhy/6rTSFbzwKB0Jm
        ebwzQfwjQY6f55MiI/RqDCyuPj3r3jyVRkK86pQKBAJwFHyqj9KaKXMZjfVnowLh
        9svIGfNbGHpucATqREvUHuQbNnqkCx8VVhtYkhDb9fEP2xBu5VvHbR+3nfVhMut5
        G34Ct5RS7Jt6LIfFdtcn8CaSas/l1HbiGeRgc70X/9aYx/V/CEJv0lIe8gP6uDoW
        FPIZ7d6vH+Vro6xuWEGiuMaiznap2KhZmpkgfupyFmplh0s6knymuQINBFit2ioB
        EADneL9S9m4vhU3blaRjVUUyJ7b/qTjcSylvCH5XUE6R2k+ckEZjfAMZPLpO+/tF
        M2JIJMD4SifKuS3xck9KtZGCufGmcwiLQRzeHF7vJUKrLD5RTkNi23ydvWZgPjtx
        Q+DTT1Zcn7BrQFY6FgnRoUVIxwtdw1bMY/89rsFgS5wwuMESd3Q2RYgb7EOFOpnu
        w6da7WakWf4IhnF5nsNYGDVaIHzpiqCl+uTbf1epCjrOlIzkZ3Z3Yk5CM/TiFzPk
        z2lLz89cpD8U+NtCsfagWWfjd2U3jDapgH+7nQnCEWpROtzaKHG6lA3pXdix5zG8
        eRc6/0IbUSWvfjKxLLPfNeCS2pCL3IeEI5nothEEYdQH6szpLog79xB9dVnJyKJb
        VfxXnseoYqVrRz2VVbUI5Blwm6B40E3eGVfUQWiux54DspyVMMk41Mx7QJ3iynIa
        1N4ZAqVMAEruyXTRTxc9XW0tYhDMA/1GYvz0EmFpm8LzTHA6sFVtPm/ZlNCX6P1X
        zJwrv7DSQKD6GGlBQUX+OeEJ8tTkkf8QTJSPUdh8P8YxDFS5EOGAvhhpMBYD42kQ
        pqXjEC+XcycTvGI7impgv9PDY1RCC1zkBjKPa120rNhv/hkVk/YhuGoajoHyy4h7
        ZQopdcMtpN2dgmhEegny9JCSwxfQmQ0zK0g7m6SHiKMwjwARAQABiQQ+BBgBCAAJ
        BQJYrdoqAhsCAikJEI2BgDwOv82IwV0gBBkBCAAGBQJYrdoqAAoJEH6gqcPyc/zY
        1WAP/2wJ+R0gE6qsce3rjaIz58PJmc8goKrir5hnElWhPgbq7cYIsW5qiFyLhkdp
        YcMmhD9mRiPpQn6Ya2w3e3B8zfIVKipbMBnke/ytZ9M7qHmDCcjoiSmwEXN3wKYI
        mD9VHONsl/CG1rU9Isw1jtB5g1YxuBA7M/m36XN6x2u+NtNMDB9P56yc4gfsZVES
        KA9v+yY2/l45L8d/WUkUi0YXomn6hyBGI7JrBLq0CX37GEYP6O9rrKipfz73XfO7
        JIGzOKZlljb/D9RX/g7nRbCn+3EtH7xnk+TK/50euEKw8SMUg147sJTcpQmv6UzZ
        cM4JgL0HbHVCojV4C/plELwMddALOFeYQzTif6sMRPf+3DSj8frbInjChC3yOLy0
        6br92KFom17EIj2CAcoeq7UPhi2oouYBwPxh5ytdehJkoo+sN7RIWua6P2WSmon5
        U888cSylXC0+ADFdgLX9K2zrDVYUG1vo8CX0vzxFBaHwN6Px26fhIT1/hYUHQR1z
        VfNDcyQmXqkOnZvvoMfz/Q0s9BhFJ/zU6AgQbIZE/hm1spsfgvtsD1frZfygXJ9f
        irP+MSAI80xHSf91qSRZOj4Pl3ZJNbq4yYxv0b1pkMqeGdjdCYhLU+LZ4wbQmpCk
        SVe2prlLureigXtmZfkqevRz7FrIZiu9ky8wnCAPwC7/zmS18rgP/17bOtL4/iIz
        QhxAAoAMWVrGyJivSkjhSGx1uCojsWfsTAm11P7jsruIL61ZzMUVE2aM3Pmj5G+W
        9AcZ58Em+1WsVnAXdUR//bMmhyr8wL/G1YO1V3JEJTRdxsSxdYa4deGBBY/Adpsw
        24jxhOJR+lsJpqIUeb999+R8euDhRHG9eFO7DRu6weatUJ6suupoDTRWtr/4yGqe
        dKxV3qQhNLSnaAzqW/1nA3iUB4k7kCaKZxhdhDbClf9P37qaRW467BLCVO/coL3y
        Vm50dwdrNtKpMBh3ZpbB1uJvgi9mXtyBOMJ3v8RZeDzFiG8HdCtg9RvIt/AIFoHR
        H3S+U79NT6i0KPzLImDfs8T7RlpyuMc4Ufs8ggyg9v3Ae6cN3eQyxcK3w0cbBwsh
        /nQNfsA6uu+9H7NhbehBMhYnpNZyrHzCmzyXkauwRAqoCbGCNykTRwsur9gS41TQ
        M8ssD1jFheOJf3hODnkKU+HKjvMROl1DK7zdmLdNzA1cvtZH/nCC9KPj1z8QC47S
        xx+dTZSx4ONAhwbS/LN3PoKtn8LPjY9NP9uDWI+TWYquS2U+KHDrBDlsgozDbs/O
        jCxcpDzNmXpWQHEtHU7649OXHP7UeNST1mCUCH5qdank0V1iejF6/CfTFU4MfcrG
        YT90qFF93M3v01BbxP+EIY2/9tiIPbrd
        =0YYh
        -----END PGP PUBLIC KEY BLOCK-----

snap:
  commands:
    - [remove, lxd]
    - [remove, core20]
    - [remove, snapd]

packages:
  - docker-ce
  - docker-ce-cli
  - containerd.io
  - docker-compose-plugin

```

My docker info looks like this
```
Client:
 Context:    default
 Debug Mode: false
 Plugins:
  app: Docker App (Docker Inc., v0.9.1-beta3)
  buildx: Docker Buildx (Docker Inc., v0.9.1-docker)
  compose: Docker Compose (Docker Inc., v2.12.2)
  scan: Docker Scan (Docker Inc., v0.21.0)

Server:
 Containers: 2
  Running: 0
  Paused: 0
  Stopped: 2
 Images: 3
 Server Version: 20.10.21
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 1
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 io.containerd.runtime.v1.linux runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 1c90a442489720eec95342e1789ee8a5e1b9536f
 runc version: v1.1.4-0-g5fd4c4d
 init version: de40ad0
 Security Options:
  apparmor
  seccomp
   Profile: default
 Kernel Version: 5.4.0-131-generic
 Operating System: Ubuntu 20.04.5 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 6
 Total Memory: 3.839GiB
 Name: ubuntu
 ID: FYGU:QQOP:NXE3:3CPX:DX2X:UHWR:3HFK:2A63:YQYT:N3QQ:WWOS:HNVA
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

WARNING: No swap limit support
```

</details>

**Comment by chrism** [comment]
Hi @vitoyucepi – sorry it took me a while to get back to you.

I just did a `docker info` on my instance and here is the output... I think you are right it seems like it is `20.10.7`

```
Client:
 Context:    default
 Debug Mode: false
 Plugins:
  app: Docker App (Docker Inc., v0.9.1-beta3)
  buildx: Build with BuildKit (Docker Inc., v0.5.1-docker)
  scan: Docker Scan (Docker Inc., v0.8.0)

Server:
 Containers: 0
  Running: 0
  Paused: 0
  Stopped: 0
 Images: 2
 Server Version: 20.10.7
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 1
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: runc io.containerd.runc.v2 io.containerd.runtime.v1.linux
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: d71fcd7d8303cbf684402823e425e9dd2e99285d
 runc version: b9ee9c6314599f1b4a7f497e1f1f856fe433d3b7
 init version: de40ad0
 Security Options:
  apparmor
  seccomp
   Profile: default
 Kernel Version: 5.4.0-77-generic
 Operating System: Ubuntu 20.04.2 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 1
 Total Memory: 981.2MiB
 Name: dream-radio-player
 ID: ZQU2:Z6S7:7G6D:6EUZ:FFNF:BXOJ:XF4I:BG5G:W4QU:3MQF:SIBG:FH3F
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
```

Let me know if you want me to check anything else.

Thanks!

**Comment by vitoyucepi** [comment]
Hi @chrism,
The next step is to find out how to update it.
`apt-cache policy docker-ce` will print all versions of *docker-ce* in the system package manager cache.
I'm assuming your docker installed using apt.

**Comment by chrism** [comment]
Seems like it must have been @vitoyucepi because this is the output...

```
docker-ce:
  Installed: 5:20.10.7~3-0~ubuntu-focal
  Candidate: 5:20.10.21~3-0~ubuntu-focal
  Version table:
     5:20.10.21~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:20.10.20~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:20.10.19~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:20.10.18~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:20.10.17~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:20.10.16~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:20.10.15~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:20.10.14~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:20.10.13~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:20.10.12~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:20.10.11~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:20.10.10~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:20.10.9~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:20.10.8~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
 *** 5:20.10.7~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
        100 /var/lib/dpkg/status
     5:20.10.6~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:20.10.5~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:20.10.4~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:20.10.3~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:20.10.2~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:20.10.1~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:20.10.0~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:19.03.15~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:19.03.14~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:19.03.13~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:19.03.12~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:19.03.11~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:19.03.10~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
     5:19.03.9~3-0~ubuntu-focal 500
        500 https://download.docker.com/linux/ubuntu focal/stable amd64 Packages
```

**Comment by vitoyucepi** [comment]
I think you should consider doing an update.
I recommend `apt-get update && apt-get dist-upgrade` to get newer versions of all packages.
If you'd like to update only docker, then `apt-get install docker-ce docker-ce-cli containerd.io docker-compose-plugin` will do the trick.
This command should print something like
```
The following packages will be upgraded:
  docker-ce docker-ce-cli containerd.io docker-compose-plugin
```

Maybe even do a `do-release-upgrade` to upgrade to `ubuntu:22.04`.

@chrism, you should remember that these changes could potentially break something. I'm not aware of your system setup, but doing a full system backup beforehand would be a great idea.
Digital Ocean has a backup function to do that.

**Comment by chrism** [comment]
Thank you so much @vitoyucepi its too late for me to do this tonight but I will try tomorrow.

Really appreciate your advice.

**Comment by chrism** [comment]
TL;DR Updating the Digital Ocean droplet to Docker `20.10.21` resolved this issue and I am now running `2.1.2` in production.

I am not sure but maybe @toots it is worth documenting that while `2.1.0` supported Docker `20.10.7`, `2.1.2` no longer does (unless I have missed it and this is already documented)?

Once again thank you so much for your help @vitoyucepi, I followed your advice and updated using `apt-get update && apt-get dist-upgrade`.

This successfully updated Docker to `20.10.21` and then just updating `docker-compose.yml` to the latest liquidsoap version was enough to make it work.

<details>
<summary>Full output of docker info after updating</summary>
<pre>
Client:
 Context:    default
 Debug Mode: false
 Plugins:
  app: Docker App (Docker Inc., v0.9.1-beta3)
  buildx: Docker Buildx (Docker Inc., v0.9.1-docker)
  scan: Docker Scan (Docker Inc., v0.21.0)

Server:
 Containers: 2
  Running: 0
  Paused: 0
  Stopped: 2
 Images: 2
 Server Version: 20.10.21
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 1
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 io.containerd.runtime.v1.linux runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 770bd0108c32f3fb5c73ae1264f7e503fe7b2661
 runc version: v1.1.4-0-g5fd4c4d
 init version: de40ad0
 Security Options:
  apparmor
  seccomp
   Profile: default
 Kernel Version: 5.4.0-77-generic
 Operating System: Ubuntu 20.04.5 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 1
 Total Memory: 981.2MiB
 Name: dream-radio-player
 ID: ZQU2:Z6S7:7G6D:6EUZ:FFNF:BXOJ:XF4I:BG5G:W4QU:3MQF:SIBG:FH3F
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
</pre>
</details>

**Comment by vitoyucepi** [comment]
Hi @chrism,
This is a container engine problem. You can still use modern images on older docker by providing the command line option `--security-opt seccomp=unconfined` or setting `security_opt: [seccomp:unconfined]` in the docker-compose file.
For example, I've found an article about the problem https://medium.com/nttlabs/1cd439d9921.

P.S. Ubuntu 20.04 support will end in 2.5 years. Use Ubuntu 22.04 as a virtual machine template.

**Comment by mcfiredrill** [comment]
Thanks for posting this, I had the same issue. Upgrading docker solved it.
